src/modules/rlm_pap/rlm_pap.c

```diff
@@ -375,11 +401,19 @@ static rlm_rcode_t CC_HINT(nonnull) mod_authorize(void *instance, REQUEST *reque
 
 #ifdef HAVE_OPENSSL_EVP_H
 		case PW_SHA2_PASSWORD:
+		case PW_SSHA224_PASSWORD:
+		case PW_SSHA256_PASSWORD:
+		case PW_SSHA384_PASSWORD:
+		case PW_SSHA512_PASSWORD:
```

Comment:
Is there a standard you would prefer for each of the normalisable formats to place delimiters in binary blobs? It was because of this that I only implemented parsing the PBKDF2 as a base64 encoded string.